src/documentation/LoginPage/README.mdx

```diff
@@ -3,3 +3,7 @@ The **Login** page is based on the Modal component. Only use it for external log
 ### Related
 
 [Modal](?path=/docs/components-modal--modal-regular)
+
+## React Sourcecode
```

Comment:
Maybe we can find a way to automate the Messages and links to Github in the future.

Reply:
sure